drivers/input/touchscreen/edt-ft5x06.c

```diff
@@ -47,16 +40,32 @@
 #define WORK_REGISTER_NUM_X		0x33
 #define WORK_REGISTER_NUM_Y		0x34
 
+#define PMOD_REGISTER_ACTIVE		0x00
+#define PMOD_REGISTER_HIBERNATE		0x03
+
 #define M09_REGISTER_THRESHOLD		0x80
 #define M09_REGISTER_GAIN		0x92
 #define M09_REGISTER_OFFSET		0x93
 #define M09_REGISTER_NUM_X		0x94
 #define M09_REGISTER_NUM_Y		0x95
 
+#define M09_ID_G_MODE			0xa4
+#define M09_ID_G_MODE_POLL			0x00
+#define M09_ID_G_MODE_IRQ			0x01
+
+#define M09_ID_G_PMODE			0xa5
+#define M09_ID_G_PMODE_ACTIVE			0x00
+
+#define EV_REGISTER_THRESHOLD		0x40
+#define EV_REGISTER_GAIN		0x41
+#define EV_REGISTER_OFFSET_Y		0x45
+#define EV_REGISTER_OFFSET_X		0x46
+
 #define NO_REGISTER			0xff
 
 #define WORK_REGISTER_OPMODE		0x3c
 #define FACTORY_REGISTER_OPMODE		0x01
+#define PMOD_REGISTER_OPMODE		0xa5
 
 #define TOUCH_EVENT_DOWN		0x00
 #define TOUCH_EVENT_UP			0x01
```

Comment:
Please check the alignment. I think tabs and spaces were mixed

Reply:
These changes come from the original driver, changing the original driver code too much is a bad idea, when we need to upgrade the Kernel or driver this can cause unwanted problems.